app/assets/javascripts/discourse/app/components/reviewable-item.hbs

```diff
@@ -37,10 +37,7 @@
         {{#if this.reviewable.created_by}}
           <div class="editable-created-by">
             {{avatar this.reviewable.created_by imageSize="tiny"}}
-            <ReviewableCreatedByName
-              @user={{this.reviewable.created_by}}
-              @tagName=""
-            />
+            <ReviewableCreatedByName @user={{this.reviewable.created_by}} />
```

Comment:
btw, possible followups like this:

* https://github.com/discourse/discourse-akismet/blob/main/assets/javascripts/discourse/templates/components/reviewable-akismet-post.hbs#L9C18-L9C18
* and one `{{reviewable-created-by-name` in a custom plugin

Reply:
yes I have seen it, not sure how important it is to fix this, technically it's just a noop now